fuzzing/stun_fuzzer.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package fuzzing
+
+import "tailscale.com/net/stun"
+
+func FuzzStunParser(data []byte) int {
+	_, _, _, err := stun.ParseResponse(data)
+	if err != nil {
+		return 0
+	}
```

Comment:
Try stun.ParseBindingRequest here as well?

(And then rename this just Fuzz, and don't worry about returning 0 vs 1. The signal from those is weak, and go-fuzz's support for running multiple fuzz functions is poor enough that it's nicer to just have a single mixed fuzz function.)